libnetwork/ipam/allocator.go

```diff
@@ -204,7 +204,7 @@ func (a *Allocator) retrieveBitmask(k SubnetKey, n *net.IPNet) (*bitseq.Handle,
 	bm, ok := a.addresses[k]
 	a.Unlock()
 	if !ok {
-		logrus.Debugf("Retrieving bitmask (%s, %s)", k.String(), n.String())
+		logrus.Debugf("Retrieving bitmask (%s, %s)", k, n)
```

Comment:
Out of curiosity; would `.WithFields()` work for these, or would that reintroduce the overhead?

(For some logs, I even wondered if we need to condor using `.Tracef()` level)

Reply:
`.WithFields()` would reintroduce the overhead